File: Scripts/laptop/factorial/php/factorial.php
<?php
function factorial($n) {
  if ($n < 0) {
    return 0;
  }

  if ($n == 0) {
    return 1;
  }

  else {
    return $n * factorial($n-1);
  }
}


$r = 1;

function executeTask($i) {

	factorial(10);
	return $i + 1;
}

for ( $i = 0; $i < 2000000000; $i++) {
	$r = executeTask($i + $r);
}

?>

File: Scripts/laptop/factorial/r/factorial.r
fact <- function(n) {
  if ( n <= 1 ) 1
  else n * fact(n-1)
}

executeTask <- function(i) {
	fact(10)
	return(i + 1)
}

r = 1

for(i in 0:2000000000) {
	r = executeTask(i + r)
}
